Prefabs/Interactions/Controllables/SharedResources/Scripts/RotationalDrive.cs

```diff
@@ -67,5 +111,97 @@ protected override float CalculateValue(DriveAxis.Axis driveAxis, FloatRange lim
 
             return Mathf.Clamp(pseudoRotation, limits.minimum, limits.maximum);
         }
+
+        /// <summary>
+        /// Attempts to retrieve a simple x, y or z euler angle from the <see cref="Transform.localEulerAngles"/> utilizing any other axis rotation.
+        /// </summary>
+        /// <returns>The actual axis angle from 0f to 360f.</returns>
+        protected virtual Vector3 GetSimpleEulerAngles()
+        {
+            Vector3 currentEulerAngle = GetDriveTransform().localEulerAngles;
+            if (facade.DriveAxis == DriveAxis.Axis.XAxis && !currentEulerAngle.y.ApproxEquals(0f, 1f))
+            {
+                currentEulerAngle.x = currentEulerAngle.y - (currentEulerAngle.x > (circleDegrees * 0.5f) ? currentEulerAngle.x - circleDegrees : currentEulerAngle.x);
+            }
+            return currentEulerAngle;
+        }
+
+        /// <summary>
+        /// Calculates a multiplier based on the direction the rotation is travelling.
```

Comment:
`traveling`

Reply:
https://dictionary.cambridge.org/dictionary/english/travelling

Reply:
>us usually traveling

The British English strikes again 😛

Reply:
stupid spellings